packages/datadog-plugin-aws-sdk/src/services/bedrockruntime/utils.js

```diff
@@ -176,11 +191,13 @@ function extractRequestParams (params, provider) {
   }
 }
 
-function extractTextAndResponseReason (response, provider, modelName, shouldSetChoiceIds) {
+function extractTextAndResponseReason (response, provider, modelName, shouldSetChoiceIds = undefined) {
   const body = JSON.parse(Buffer.from(response.body).toString('utf8'))
-
+  if (shouldSetChoiceIds === undefined) {
```

Comment:
```suggestion
  if (!shouldSetChoiceIds) {
```
if taking the above suggestion as well. not sure if `shouldSetChoiceIds` is used elsewhere in this method too, but if so it should also change accordingly